packages/flutter/lib/src/material/text_field.dart

```diff
@@ -432,7 +432,8 @@ class TextField extends StatefulWidget {
     return enableInteractiveSelection ?? !obscureText;
   }
 
-  /// Called when the user taps on this text field.
+  /// {@template flutter.material.textfield.onTap}
+  /// Called for each distinct tap except for every second tap of a double tap.
```

Comment:
I guess this is an undesirable side effect of using `_handleSingleTapUp`?

Reply:
I think so, that sounds familiar.  We could hack in the second taps but it's probably fine if no one is asking for it.

Reply:
It looks that way, yes. Although it's a little quirky, I think this is what most developers want from this callback.